frontend/src/lib/navigation.ts

```diff
@@ -1,4 +1,12 @@
-import { goto } from '$app/navigation'
+import { goto as svelteGoto } from '$app/navigation'
+import { base } from '$lib/navigation'
+
+export { base } from '$app/paths'
+
+export function goto(path, options = {}) {
+	const fullPath = path.startsWith(base) ? path : `${base}${path}`;
```

Comment:
optimization since this is now used everywhere

have a global const
export const CUSTOM_BASE = base && base != ""

then modify this function to be:
if (!CUSTOM_BASE) { return svelteGoto(path, options) }

It's nit but easy to optimize